src/pydap/cas/get_cookies.py

```diff
@@ -24,7 +24,10 @@ def setup_session(uri,
     '''
 
     if session is None:
-        headers = [('User-agent', pydap.lib.__version__)]
+        # Connections must be closed since some CAS
+        # will cough when connections are kept alive:
+        headers = [('User-agent', pydap__version__),
```

Comment:
Should this be ``'Pydap/{}'.format(pydap__version__)``, e.g. "Pydap/3.2.0" to conform to the [user-agent RFC](http://www.ietf.org/rfc/rfc2616.txt) (see section 3.8).